linode/lke/cluster.go

```diff
@@ -338,7 +337,7 @@ func matchPoolsWithSchema(pools []linodego.LKENodePool, declaredPools []interfac
 				continue
 			}
 
-			if !slices.Equal(helper.ExpandStringSet(declaredPool["tags"].(*schema.Set)), apiPool.Tags) {
+			if helper.CompareStringSets(helper.ExpandStringSet(declaredPool["tags"].(*schema.Set)), apiPool.Tags) {
```

Comment:
I think there might be a missing `!` (not) here 👀

Reply:
Ah sorry, just added. Thanks!